superset/assets/javascripts/SqlLab/components/ResultSet.jsx

```diff
@@ -197,20 +201,6 @@ class ResultSet extends React.PureComponent {
             </div>
           </div>
         );
-      } else if (query.resultsKey) {
-        if (results && data && data.length === 0) {
-          // if fetched result contains zero rows of data
-          return <Alert bsStyle="warning">The asynchronous query returned no data</Alert>;
```

Comment:
It will go :[https://github.com/vera-liu/caravel/blob/676f4f27eb8fabc63da2f93e1f91c69aff716ca9/superset/assets/javascripts/SqlLab/components/ResultSet.jsx#L217](url)